source/layouts/layout.erb

```diff
@@ -112,8 +112,14 @@ under the License.
               Tutorials
```

Comment:
This part of the menu should be removed I think?

Reply:
Done !